pkg/interface/chat/src/js/components/chat.js

```diff
@@ -89,21 +96,21 @@ export class ChatScreen extends Component {
       props.envelopes.length >= prevProps.envelopes.length + 10
     ) {
       this.hasAskedForMessages = false;
+    } else if(props.length !== prevProps.length &&
+       prevProps.length === prevProps.read
+    ) {
+      this.setState({ read: props.length })
+      this.props.api.chat.read(this.props.station);
     }
 
-    // FF logic
     if (
-      navigator.userAgent.includes("Firefox") &&
       (props.length !== prevProps.length ||
        props.envelopes.length !== prevProps.envelopes.length ||
        getNumPending(props) !== this.lastNumPending ||
        state.numPages !== prevState.numPages)
     ) {
-      if(state.scrollBottom) {
-        setTimeout(() => {
-          this.scrollToBottom();
-        })
-      } else {
+      this.scrollToBottom();
```

Comment:
here we try to scrollToBottom upon loading any new messages. Safari not scrolling to bottom on load is a result of a race condition where we try to scroll to the bottom before any messages load in. 
@matildepark